engine/graphics/src/graphics.h

```diff
@@ -293,6 +293,50 @@ namespace dmGraphics
         uint8_t       : 5;
     };
 
+    const static uint8_t DM_GRAPHICS_STATE_WRITE_R = 0x1;
+    const static uint8_t DM_GRAPHICS_STATE_WRITE_G = 0x2;
+    const static uint8_t DM_GRAPHICS_STATE_WRITE_B = 0x4;
+    const static uint8_t DM_GRAPHICS_STATE_WRITE_A = 0x8;
+
+    struct PipelineState
```

Comment:
This could potentially be ever so slightly optimized, the original pipeline was a union between all these bits and a 64 bit uint so we could do a fast compare between pipelines. But we couldn't fit all features into that representation and the comparison was never used anyway